### PR TITLE
Vet clinic

### PR DESCRIPTION
**Pull request by Isabellanamutenda**
Join tables 
In this project I:

- Created vets table
- Added data to the table
- Wrote queries

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,177 +1,38 @@
...
+UPDATE animals SET owner_id=4 WHERE name IN ('Charmander','Squirtle','Blossom');
+UPDATE animals SET owner_id=5 WHERE name IN ('Angemon','Boarmon');
+
+INSERT INTO vets (id,name,age,date_of_graduation) VALUES (1, 'William Tatcher', 45, '2000-04-23');
+INSERT INTO vets (id,name,age,date_of_graduation) VALUES (2, 'Maisy Smith', 26, '2019-01-17');
+INSERT INTO vets (id,name,age,date_of_graduation) VALUES (3, 'Stephanie Mendez', 64, '1981-06-08');
+INSERT INTO vets (id,name,age,date_of_graduation) VALUES (4, 'Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations (vets_id,species_id) VALUES (1,1),(3,2),(3,1),(4,2);
+
```
Comment (Emmyn5600) [inline data.sql]: - [x] You are missing some information inside the `specializations` table.

File: data.sql [hunk 2]
```diff
@@ -1,177 +1,38 @@
...
+INSERT INTO visits (animals_id,vets_id,date) VALUES (1, 1, '2020-05-24'),(1, 3, '2020-07-22'),(2, 4, '2021-02-02'),(3, 2, '2020-01-05');
+INSERT INTO visits (animals_id,vets_id,date) VALUES (3, 2, '2020-03-08'),(3, 2, '2020-05-14'),(4, 2, '2021-05-04'),(5, 4, '2021-02-24');
+INSERT INTO visits (animals_id,vets_id,date) VALUES (6, 2, '2019-12-21'),(6, 1, '2020-08-10'),(6, 2, '2021-04-07'),(7, 3, '2019-09-29');
+INSERT INTO visits (animals_id,vets_id,date) VALUES (8, 4, '2020-10-03'),(8, 4, '2020-11-04'),(9, 2, '2019-01-24'),(9, 2, '2019-05-15');
+INSERT INTO visits (animals_id,vets_id,date) VALUES (9, 2, '2020-02-27'),(9, 2, '2020-08-03'),(10, 3, '2020-05-24'),(10, 1, '2021-01-11');
```
Comment (Emmyn5600) [inline data.sql]: - [x] Even here, You are missing some information inside the `visits` table.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi, @Isabellanamutenda 

Good job so far! 👏🏼 

Great work on making the project requirements:clap:
You've done well implementing the project requirements, but there are still some issues which aren't been addressed yet 
 
## STATUS: CHANGES REQUESTED :recycle:


### Changes requested.

- [ ] Check the comment under the review.
- [ ] You can also check out the `queries.sql` to modify some implementation

 #### Optional suggestions
 
 - N/A
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy: There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**